op-deployer/pkg/deployer/opcm/bootstrap_test.go

```diff
@@ -0,0 +1,32 @@
+package opcm_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/foundry"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/script"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+// createTestHost is a helper function for testing deploy script wrappers
+func createTestHost(t *testing.T) *script.Host {
+	// Create a logger
+	logger, _ := testlog.CaptureLogger(t, log.LevelInfo)
+
+	// Create an artifact filesystem pointing to the bedrock contracts artifact directory
+	af := foundry.OpenArtifactsDir("../../../../packages/contracts-bedrock/forge-artifacts")
```

Comment:
This forge-artifacts dir is gitignored. Will this test fail if it runs before the contracts are compiled (and populate this dir)? Maybe better to use a diff dir here?

Reply:
In the CI we build the contracts before running this so that's not an issue. I will check whether we need to add a `just` target dependency - the whole point of this test is to test against the current version of the contract, not a fixture

Reply:
I added a `test` target to just file that will first build the contracts, then run the tests